src/pyskindose/plotting/create_plot_and_save_to_file.py

```diff
@@ -20,4 +18,5 @@ def create_plot_and_save_to_file(mode: str, data: List[Union[go.Mesh3d, go.Scatt
     logger.debug(f"Creating plot and savint to file {plot_filename}")
 
     fig = go.Figure(data=data, layout=layout)
```

Comment:
"Go figure!" :P

Reply:
?

Reply:
Just a joke. I saw the line `fig = go.Figure(...)` and thought of the expression _Go figure!_ :)